requirements.txt

```diff
@@ -1,21 +0,0 @@
-apiai==1.2.3
```

Comment:
please don't delete this file, it is necessary. It's because of this the checks are failing